### PR TITLE
Add properties to view recording screen

### LINKED ISSUE #37: Show metadata on view recordings page
As per previous UI

### PR DESCRIPTION
**Pull request by arthurmcgregor**
Closes #37
- Add recording properties similar to old UI
- Plus improve battery icon on query recordings table
- Display of 'Relative to Dusk/Dawn' could be improved -> I wasn't sure what units were here

I only have a limited set of recordings to look at so once again please let me know if anything odd comes up!

### CODE REVIEW

File: src/components/BatteryLevel.vue [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+};
+</script>
+
+<style scoped>
+  .fa-spin {
+    color: #f5f5f5;
+    margin: 1rem auto;
+    display: flex;
+  }
+</style>
```
Comment (mjs) [inline src/components/BatteryLevel.vue]: Love this!

File: src/components/Video/VideoProperties.vue [hunk 2]
```diff
@@ -1,12 +1,31 @@
...
+          </p>
+          <p v-else-if="recording.additionalMetadata && prop.key === 'additionalMetadata'">
+            <strong>Additional Metadata:</strong>
+            <ul>
+              <li
+                v-for="(key, index) of Object.keys(recording['additionalMetadata'])"
+                :key="index">
+                {{ key }}: {{ recording['additionalMetadata'][key] }}
+              </li>
+            </ul>
```
Comment (mjs) [inline src/components/Video/VideoProperties.vue]: This is fine but I wonder if it would look better as a two column table. Your call though.

**Comment by mjs** [comment]
I've tried it out with a variety of recordings and it looks great.

File: src/components/Video/VideoProperties.vue (hunk 2 above)
Comment (arthurmcgregor) [reply src/components/Video/VideoProperties.vue]: I've implemented that - see what you think

**Comment by mjs** [comment]
@arthurmcgregor looks like there's a conflict

**Pull request merged**